src/libs/constants/random.ts

```diff
@@ -0,0 +1,2 @@
+export const SUPPORT_EMAIL = "support.black.circle@gmail.com";
+export const FIRST_INDEX = 0;
```

Comment:
Separate those const into two different files and name the file the same as the constant please.

Reply:
oh I would hate my work if I would work on on the project where are the files for each constant) I don't like name - random, I think it could be just index.ts. And I don't see why we need constant FIRST_INDEX as its always 0 in the programming